app/assets/stylesheets/active_admin/components/_flash_messages.scss

```diff
@@ -1,23 +1,18 @@
 body.logged_in {
   .flash {
-    @include gradient(#f7f1d3, #f5edc5);
-    @include text-shadow(#fafafa);
-    border-bottom: 1px solid #eee098;
+    background-color: #f7f1d3;
```

Comment:
Color literals like `#f7f1d3` should only be used in variable declarations; they should be referred to via variable everywhere else.